.github/workflows/flatpak.yaml

```diff
@@ -0,0 +1,23 @@
+on: tag
+
+jobs: 
+  build: flatpack
+  - uses: actions/checkout@v2   # automatically clone the repository
+    runs-on: ubuntu-20.04  # verified on ubuntu-20.04 and should run on any debian distro
```

Comment:
In this file, the `runs-on` directive sets the platform to use for the workflow. The platforms available are [listed here](https://docs.github.com/en/actions/reference/software-installed-on-github-hosted-runners) . I think the comment can be removed.

Reply:
ok, it is removed now, thank you.